### PR TITLE
Capture ministry and primary contact info

### PR DESCRIPTION
**Pull request by kamorel**
# Description
Capture ministry and primary contact information for the Acronym. 
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2472

## Types of changes

New feature (non-breaking change which adds functionality) 

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/src/migrations/20220407224032-add-acronym-ministry.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+'use strict';
```
Comment (jujaga) [inline app/src/migrations/20220407224032-add-acronym-ministry.js:1]: Don't think we explicitly need this clause in our node execution environments, but doesn't hurt either I suppose.

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://getok-dev.apps.silver.devops.gov.bc.ca/pr-366

File: app/frontend/src/components/RequestForm.vue [hunk 2]
```diff
@@ -60,13 +60,58 @@
...
+            solo
+            outlined
+          >
+            <template #append-outer />
+          </v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col>
+          <label>Contact &nbsp;</label>
```
[the remark this answers is not in the document]
Comment (TimCsaky) [reply app/frontend/src/components/RequestForm.vue:81]: 'Contact' seems ok to  me.. another option is 'Contact Person'. I wonder if the tooltip should say: 'Generally the Name AND Email of the ...'

File: app/frontend/src/components/admin/AccessGrant.vue [hunk 3]
```diff
@@ -45,6 +45,38 @@
                 outlined
               />
             </v-col>
+            <v-col sm="6" lg="4" class="pt-2">
+              <label>Ministry</label>
+              <v-text-field
+                v-model="ministry"
+                dense
+                flat
+                hint="Acronym of the ministry this new Acronym is under"
```
Comment (TimCsaky) [inline app/frontend/src/components/admin/AccessGrant.vue:54]: minor point.. maybe this should read: 'Ministry for client application (eg: NRIDS)'

Comment (jujaga) [reply app/frontend/src/components/admin/AccessGrant.vue:54]: That or "Ministry this application is under"

File: app/frontend/src/components/RequestForm.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (TimCsaky) [reply app/frontend/src/components/RequestForm.vue:81]: having seen the rest of the code.. maybe this is fine as is.

**Review by jujaga — APPROVED** [review]

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://getok-dev.apps.silver.devops.gov.bc.ca/pr-366

**Comment by TimCsaky** [comment]
looks greaat! glad you found your way around the code.. there's a lot going on with this app in my view.
As mentioned, you can use the vuetify 'pr-3' class but that's optional.
now i need to go and remove all the test service clients i created in keycloak 🤣

**Review by TimCsaky — APPROVED** [review]

**Pull request merged**